proxy.go

```diff
@@ -321,24 +321,25 @@ func realMain() {
 
```

Comment:
Just a query - just above here, if we fail to set up the logger - we log it :-/, and then do not quit - do we think that is the right thing to do? The only alternative being to quit at that point?

Reply:
Yep - this was missing exit call which I've now added.

I can see that reading this side of the code still looks odd, but it's correct since even before `setupLogger()` gets called, we have valid logger - `setupLogger()` simply sets up (the existing) logger some more ;) Hence, logging the failure to setup the logger is correct.